docker-for-mac/troubleshoot.md

```diff
@@ -352,3 +353,91 @@ in the Apple documentation, and Docker Desktop [Mac system requirements](install
+  * Enabling virtualization in BIOS
+  * Enabling Windows features
+  * Genuine installation crashes
+  * Crashes due to conflicts with other software
+  * Installation issues from enterprise deployment
+* First run issues
+   * Failure to launch Docker Desktop on first run
+* Usage issues
+  * Crash closing software
+  * Docker Desktop Not behaving as expected
+* Configuration issues
+* Basic product ‘how to’ questions such as ‘how to work efficiently on WSL 2’
+
+### What is not supported?
+
+Docker Desktop excludes support for the following types of issues:
+
+* Use on or in conjunction with hardware or software other than that specified in the applicable documentation
+* Running on unsupported operating systems, including beta/preview versions of operating systems
+* Use of Docker/support for learning or fixing Docker commands
+* Support for the Docker engine, Docker CLI, or other bundled Linux components
+* Support for Kubernetes
+* Features labeled as experimental
+* System/Server administration activities
+* Supporting Desktop as a production runtime
+* Scale deployment/multi-machine installation of Desktop
+* Routine product maintenance (data backup, cleaning disk space and configuring log rotation)
+* Third-party applications not provided by Docker
+* Altered or modified Docker Software
+* Defects in the Docker Software due to hardware malfunction, abuse or improper use
+* Any version of the Docker software other than the latest version
+* Reimbursing and expenses spent for third-party services not provided by Docker
+* Docker Support excludes training, customization and integration
+
+### What versions are supported?
+
+We currently only offer support for the latest version of Docker Desktop. If you are running an older version, you may be asked to upgrade before reviewing your support request.
+
+### Can I deploy Docker Desktop to my team?
+
+You can deploy Docker Desktop to your team as part of the Team Docker subscription.
+Under the standard Desktop agreement, users need to explicitly accept the terms and these cannot be accepted on behalf of the user. With the Team plan we provide you with the rights to distribute Docker Desktop to your team.
+
+### How many machines can I get support for Docker Desktop on?
+
+As a Pro user you can get support for Docker Desktop on a single machine.
+As a Team, you can get support for Docker Desktop for the number of machines equal to the number of seats as part of your plan.
+
+### What OS’s are supported?
+
+Docker Desktop is available for Mac and Windows. The supported version information can be found on the following pages:
+
+* Mac system requirements](../docker-for-mac/install.md#system-requirements)
+* [Windows system requirements](../docker-for-windows/install.md#system-requirements)
+
+### Can I run Docker Desktop on Virtualized hardware?
+
+No, currently this is unsupported and against the terms of use.
+
+### Is Docker Desktop the same as Docker Desktop Enterprise?
```

Comment:
Let's omit this section too, it's potentially confusing. We can add it in later if necessary.